releases/links_v2.0/LINKS-make

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/make -f
+# Pipeline for the LINKS program
+# Written by Murathan Goktas
+#Default Parameters
+
+# Input Names 
+draft=draft
```

Comment:
Instead of having defaults, it would be better for the makefile to print an informative message if those required parameters aren't supplied